applications/matter_weather_station/sample.yaml

```diff
@@ -2,24 +2,26 @@ sample:
   name: Matter Weather Station
   description: Matter Weather Station application
 tests:
-  # Excluded in quarantine.yaml to limit resources usage in integration builds
-  applications.matter_weather_station.debug:
+  # Excluded in quarantine.yaml to limit resource usage in integration builds
+  # This configuration is used for testing DFU over BLE between different NCS
```

Comment:
I suggest to only remove word "testing" as it is not solely for testing, that is for a usage scenario case to support.